scripts/modified_input_file.dat

```diff
@@ -0,0 +1,14 @@
+* This is a made-up input file for the 
```

Comment:
Why is this file also checked in? Seems generated from the `base_input_file.dat`.

Reply:
Correct! .gitignore avoids *modified.*, not modified*.*, so I renamed and removed it.